ra/registration-authority.go

```diff
@@ -43,15 +45,21 @@ type RegistrationAuthorityImpl struct {
 	log         *blog.AuditLogger
 	// How long before a newly created authorization expires.
 	authorizationLifetime time.Duration
+	rlPolicies            cmd.RateLimitConfig
+	tiMu                  *sync.RWMutex
+	totalIssuedCache      *int64
```

Comment:
Ohhh you were wanting a nil check.

Hm, I'd set the lastIssuedCount into the distant past or make it the nil-able thing just to be clear.